src/manage_sql_secinfo.c

```diff
@@ -4063,17 +4086,40 @@ update_scap_cves ()
 static void
 update_scap_affected_products ()
 {
+  iterator_t cves_iter;
+  GString *cve_ids_buffer;
   g_info ("Updating affected products");
 
-  sql ("INSERT INTO scap2.affected_products"
-       "  SELECT DISTINCT scap2.cpe_match_nodes.cve_id, scap2.cpes.id"
-       "    FROM scap2.cpe_match_nodes, scap2.cpe_nodes_match_criteria,"
-       "         scap2.cpe_matches, scap2.cpes"
-       "    WHERE scap2.cpe_match_nodes.id = scap2.cpe_nodes_match_criteria.node_id"
-       "      AND scap2.cpe_nodes_match_criteria.vulnerable = 1"
-       "      AND scap2.cpe_nodes_match_criteria.match_criteria_id ="
-       "            scap2.cpe_matches.match_criteria_id"
-       "      AND scap2.cpe_matches.cpe_name_id = scap2.cpes.cpe_name_id;");
+  init_iterator(&cves_iter,
+                "SELECT DISTINCT cve_id FROM scap2.cpe_match_nodes");
+
+  int count = 0;
```

Comment:
I know it's pedantic but this breaks the pattern of putting the variable declarations all together at the top of the block, and then putting the variable definitions here below. Especially because you've mixed the two styles in the function.